test/cli/autocorrect-helpers/autocorrect-helpers.sh

```diff
@@ -0,0 +1,7 @@
+tmp="$(mktemp)"
+pre="test/cli/autocorrect-helpers/pre.rb"
+post="test/cli/autocorrect-helpers/post.rb"
+cp "$pre" "$tmp"
+main/sorbet --silence-dev-message -a "$tmp"
```

Comment:
Can you capture `stderr` here so that it shows up in `autocorrect-helpers.out`?

https://github.com/sorbet/sorbet/blob/master/test/cli/suggest-sig/suggest-sig.sh

^ also, this is a good CLI test to copy. This test uses `cat` to capture the edited file in the output, which means that any future diff can be updated by the script (e.g. right now `post.rb` must be edited by hand, or we have to record a diff in `autocorrect-helpers.out`, and diffs are trickier to read)

and it uses `mktemp -d` to get around a problem where tmpfile names show up in the `.out` output.

Reply:
Good point, done 👍

Reply:
I also added a reminder that we should clean all of this: https://github.com/sorbet/sorbet/issues/3385 since I basically copied the test from `autocorrect-abstract`.